src/services/analytics.ts

```diff
@@ -23,23 +27,20 @@ export const fetchAnalyticsData = async ({
   })
 }
 
-export const fetchLeadsAnalytics = async ({
+export const fetchLeadsImpressions = async ({
   dealerId,
   dimensions,
-  query,
   options = {},
 }: {
   dealerId: number
   dimensions?: string[]
-  query?: { verificationDateFrom: string }
   options?: ApiCallOptions
 }): Promise<Array<CockpitAnalytics>> => {
   return postData({
```

Comment:
Is this still POST?

Reply:
yes because I send the function and dimensions